### PR TITLE
Added Pizza Dough Recipe

### PR DESCRIPTION
**Pull request by Devin-Huyghebaert**

### CODE REVIEW

File: Devin-Huyghebaert.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ Ingredients
+
+    1 (.25 ounce) package active dry yeast
```
Comment (egavazzi) [inline Devin-Huyghebaert.txt:3]: You should maybe indicate the weight in grams too

**Review by egavazzi — CHANGES_REQUESTED** [review]

File: Devin-Huyghebaert.txt (hunk 1 above)
Comment (Devin-Huyghebaert) [inline Devin-Huyghebaert.txt:3]:
```suggestion
    0.8 (.2 ounce) package active dry yeast
```

**Review by AndreasKvammen — COMMENTED** [review]
looks good!

**Review by egavazzi — APPROVED** [review]
Sounds good!

**Pull request merged**